tests/integration/commands/logs/build.test.ts

```diff
@@ -0,0 +1,114 @@
+import { Mock, afterEach, beforeEach, describe, expect, test, vi } from 'vitest'
+
+import BaseCommand from '../../../../src/commands/base-command.mjs'
+import { createLogsBuildCommand } from '../../../../src/commands/logs/build.mjs'
+import { getWebSocket } from '../../../../src/utils/websockets/index.mjs'
+import { startMockApi } from '../../utils/mock-api-vitest.js'
+import { getEnvironmentVariables } from '../../utils/mock-api.mjs'
+
+vi.mock('../../../../src/utils/websockets/index.mjs', () => ({
+  getWebSocket: vi.fn(),
+}))
+
+const siteInfo = {
+  admin_url: 'https://app.netlify.com/sites/site-name/overview',
+  ssl_url: 'https://site-name.netlify.app/',
+  id: 'site_id',
+  name: 'site-name',
+  build_settings: { repo_url: 'https://github.com/owner/repo' },
+}
+
+const routes = [
+  {
+    path: 'accounts',
+    response: [{ slug: 'test-account' }],
+  },
+  {
+    path: 'sites',
+    response: [],
+  },
+  { path: 'sites/site_id', response: siteInfo },
+  { path: 'sites/site_id/service-instances', response: [] },
+  {
+    path: 'user',
+    response: { name: 'test user', slug: 'test-user', email: 'user@test.com' },
+  },
+  {
+    path: 'sites/site_id/deploys',
+    response: [
+      {
+        state: 'building',
+        context: 'production',
+        id: 'deploy-id',
+      },
+    ],
+  },
+]
+
+describe('logs:deploy command', () => {
+  //   beforeEach(() => {
```

Comment:
Does this need to be removed?

Reply:
could be left as a reminder of good times

Reply:
And by that I mean the horrible times I had getting these tests setup